testing/utils/bpcmp/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+include(ADIOSFunctions)
+
+add_executable(Test.Utils.BPcmp TestUtilsBPcmp.cpp)
+
+target_link_libraries(Test.Utils.BPcmp adios2::c adios2::cxx11)
+set(cmd_executor)
+
+add_test(NAME Utils.BPcmp
+  COMMAND ${cmd_executor} $<TARGET_FILE:Test.Utils.BPcmp>
+)
+# This test produces TestBPcmp_set1.bp and TestBPcmp_set2.bp
+
+if(UNIX)
+  set_property(TEST ${ARGS_NAME} PROPERTY
+    ENVIRONMENT
+      "PYTHONPATH=${ADIOS2_BINARY_DIR}/${CMAKE_INSTALL_PYTHONDIR}:$ENV{PYTHONPATH}"
+      "PYTHONUSERBASE=${CMAKE_BINARY_DIR}"
+  )
+else()
+  set_property(TEST ${ARGS_NAME} PROPERTY
+    ENVIRONMENT
+      "PYTHONPATH=${ADIOS2_BINARY_DIR}/${CMAKE_INSTALL_PYTHONDIR};$<SHELL_PATH:$<TARGET_FILE_DIR:adios2_py>>;$ENV{PYTHONPATH}"
+      "PATH=$<SHELL_PATH:$<TARGET_FILE_DIR:adios2_py>>;$<SHELL_PATH:$<TARGET_FILE_DIR:adios2_core>>;$ENV{PATH}"
+      "PYTHONUSERBASE=${CMAKE_BINARY_DIR}"
```

Comment:
you have to pass down these variables valeus to the bpcmp.cmake.gen and in there you have to set them for the execute_process

Reply:
Ah yes, that makes sense. I'll update this.

Reply:
Actually that is not going to work, because in the bpcmp.cmake.gen I am executing a command, not setting a test, and you do `set_property` on a test.